packages/db/index.js

```diff
@@ -83,17 +83,33 @@ async function platformaticDB (app, opts) {
     }
 
     app.log.debug({ plugin: opts.plugin }, 'loading plugin')
-    await app.register(sandbox, {
-      ...pluginOptions,
-      customizeGlobalThis (_globalThis) {
+
+    if (opts.plugin.hotReload) { // defaults to true
+      await app.register(sandbox, {
+        ...pluginOptions,
+        customizeGlobalThis (_globalThis) {
         // Taken from https://github.com/nodejs/undici/blob/fa9fd9066569b6357acacffb806aa804b688c9d8/lib/global.js#L5
-        const globalDispatcher = Symbol.for('undici.globalDispatcher.1')
-        const dispatcher = globalThis[globalDispatcher]
-        if (dispatcher) {
-          _globalThis[globalDispatcher] = dispatcher
+          const globalDispatcher = Symbol.for('undici.globalDispatcher.1')
+          const dispatcher = globalThis[globalDispatcher]
+          /* istanbul ignore else */
+          if (dispatcher) {
+            _globalThis[globalDispatcher] = dispatcher
+          }
+        }
+      })
+    } else {
+      // There is a better way then check the extension?
+      /* istanbul ignore else */
+      if (pluginOptions.path) {
+        if (pluginOptions.path.endsWith('.mjs')) {
+          const plugin = await import(pluginOptions.path)
+          await app.register(plugin, pluginOptions.options)
+        } else {
+          const plugin = require(pluginOptions.path)
+          await app.register(plugin, pluginOptions.options)
         }
```

Comment:
```suggestion
        const plugin = await import(pluginOptions.path)
        await app.register(plugin, pluginOptions.options)
```

All commonjs file could be imported... we can just user `require()`.

Reply:
Tnx! you mean `import`, obviously :)

Reply:
@mcollina had some problem with coverage (see PR description) but I think that now you can take another look.